Jenkinsfile

```
@@ -46,7 +46,7 @@ pipeline {
               checkout scm
               withEnv(["JAVA_HOME=${ tool 'jdk17' }",
                        "PATH+MAVEN=${ tool 'jdk17' }/bin:${tool 'maven3'}/bin",
-                       "MAVEN_OPTS=-Xms3072m -Xmx5120m -Djava.awt.headless=true -client -XX:+UnlockDiagnosticVMOptions -XX:GCLockerRetryAllocationCount=100"]) {
+                       "MAVEN_OPTS=-Xms3072m -Xmx5120m -Djava.awt.headless=true -client -XX:+UnlockDiagnosticVMOptions"]) {
```

```suggestion
                       "MAVEN_OPTS=-Xms3072m -Xmx5120m -Djava.awt.headless=true"]) {
```

Option `-client` does nothing, and now that the GCLocker option is gone, we don't need `UnlockDiagnosticVMOptions`

---

done